staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -271,7 +271,7 @@ func (o *CopyOptions) copyToPod(src, dest fileSpec, options *exec.ExecOptions) e
 	} else {
 		cmdArr = []string{"tar", "-xmf", "-"}
 	}
-	destDir := path.Dir(dest.File)
+	destDir := filepath.Dir(dest.File)
```

Comment:
this means we'll send a command to (presumably) a linux container containing (potentially) a windows destDir... that doesn't seem like it would work